TestRunners/TestFile.py

```diff
@@ -0,0 +1,46 @@
+from .Utils import *
+
+
+class TestFile:
+    """ represents a file to test with information on whether certain properties will be ran. 
```

Comment:
I'm not sure how to parse this, especially "will be ran". I think you mean "represents a file to test, along with information about how to run the test", but I'm not really sure.

Reply:
Shortening to "represents a file to test." When I said "with information on whether certain properties will be ran" I was referring to properties like `__INIT_TEST_CONDITIONS.TEST_WILL_RUN` in `TestFile.py` and `__INIT_PASS_TEST_CONDITIONS.IR_WILL_GENERATE` in `PassTestFile.py`, but I think it's sufficient to just say it represents a file to test